dts/arm/st/stm32f091.dtsi

```diff
@@ -5,3 +5,17 @@
  */
 
 #include <st/stm32f0.dtsi>
+
+/ {
+	soc {
+		spi2: spi@40003800 {
+			compatible = "st,stm32-spi-fifo";
+			#address-cells = <1>;
+			#size-cells = <0>;
+			reg = <0x40003800 0x400>;
+			interrupts = <26 5>;
```

Comment:
As you mention in #6238, I am just leaving a comment here to get this fixed before merging.